cmake: missing header in v1.2.3

```
In file included from pinocchio-1.2.3/bindings/python/multibody/fcl/expose-fcl.cpp:19:0:
pinocchio-1.2.3/build/include/pinocchio/bindings/python/multibody/fcl/contact.hpp:21:59: erreur fatale : pinocchio/spatial/fcl-pinocchio-conversions.hpp : Aucun fichier ou dossier de ce type
 #include "pinocchio/spatial/fcl-pinocchio-conversions.hpp"
```
It should be added here: https://github.com/stack-of-tasks/pinocchio/blob/v1.2.3/CMakeLists.txt#L160

Same problem in Kevin's system as well. There is no installation of HPP-FCL, but it is apparently not removed inside the bindings. I think the header should be conditional on the availability of Hpp-fcl. Right?

Current version: 1.2.3
System: 14.04, 64-bit

`In file included from /usr/HRP2/pinocchio/bindings/python/multibody/fcl/expose-fcl.cpp:19:0:
/usr/HRP2/pinocchio/_build_release/include/pinocchio/bindings/python/multibody/fcl/contact.hpp:21:59: fatal error: pinocchio/spatial/fcl-pinocchio-conversions.hpp: No such file or directory
 #include "pinocchio/spatial/fcl-pinocchio-conversions.hpp"`

--- a/bindings/python/CMakeLists.txt
+++ b/bindings/python/CMakeLists.txt
@@ -83,10 +83,6 @@ SET(${PROJECT_NAME}_PYTHON_HEADERS
   multibody/joint/joints-variant.hpp
   multibody/joint/joint.hpp
   multibody/joint/joint-derived.hpp
-  multibody/fcl/contact.hpp
-  multibody/fcl/collision-result.hpp
-#  multibody/fcl/distance-result.hpp
-  multibody/fcl/collision-geometry.hpp
   algorithm/algorithms.hpp 
   parsers/parsers.hpp
   )
@@ -103,7 +99,6 @@ SET(${PROJECT_NAME}_PYTHON_SOURCES
   multibody/expose-data.cpp
   multibody/expose-geometry.cpp
   multibody/joint/expose-joints.cpp
-  multibody/fcl/expose-fcl.cpp
   algorithm/expose-algorithms.cpp
   algorithm/expose-com.cpp
   algorithm/expose-kinematics.cpp
@@ -120,6 +115,7 @@ SET(${PROJECT_NAME}_PYTHON_SOURCES
   parsers/expose-parsers.cpp
   )
 
+# Parser for Python model
 SET(${PROJECT_NAME}_PARSER_PYTHON_HEADERS
   parsers/python.hpp
   )
@@ -127,6 +123,23 @@ SET(${PROJECT_NAME}_PARSER_PYTHON_HEADERS
 SET(${PROJECT_NAME}_PARSER_PYTHON_SOURCES
   parsers/python/model.cpp
   )
+
+# Python exposition of FCL
+IF(HPP_FCL_FOUND)
+  SET(${PROJECT_NAME}_PYTHON_FCL_HEADERS
+    multibody/fcl/contact.hpp
+    multibody/fcl/collision-result.hpp
+  #  multibody/fcl/distance-result.hpp
+    multibody/fcl/collision-geometry.hpp
+    )
+
+  SET(${PROJECT_NAME}_PYTHON_FCL_SOURCES
+    multibody/fcl/expose-fcl.cpp
+    )
+
+  LIST(APPEND ${PROJECT_NAME}_PYTHON_HEADERS ${${PROJECT_NAME}_PYTHON_FCL_HEADERS}) 
+  LIST(APPEND ${PROJECT_NAME}_PYTHON_SOURCES ${${PROJECT_NAME}_PYTHON_FCL_SOURCES}) 
+ENDIF(HPP_FCL_FOUND)
 
 LIST(APPEND HEADERS ${${PROJECT_NAME}_PYTHON_HEADERS})
 LIST(REMOVE_DUPLICATES HEADERS)
@@ -136,7 +149,9 @@ MAKE_DIRECTORY("${${PROJECT_NAME}_BINARY_DIR}/include/pinocchio/bindings/python"
 MAKE_DIRECTORY("${${PROJECT_NAME}_BINARY_DIR}/include/pinocchio/bindings/python/spatial")
 MAKE_DIRECTORY("${${PROJECT_NAME}_BINARY_DIR}/include/pinocchio/bindings/python/multibody")
 MAKE_DIRECTORY("${${PROJECT_NAME}_BINARY_DIR}/include/pinocchio/bindings/python/multibody/joint")
-MAKE_DIRECTORY("${${PROJECT_NAME}_BINARY_DIR}/include/pinocchio/bindings/python/multibody/fcl")
+IF(HPP_FCL_FOUND)
+  MAKE_DIRECTORY("${${PROJECT_NAME}_BINARY_DIR}/include/pinocchio/bindings/python/multibody/fcl")
+ENDIF(HPP_FCL_FOUND)
 MAKE_DIRECTORY("${${PROJECT_NAME}_BINARY_DIR}/include/pinocchio/bindings/python/parsers")
 MAKE_DIRECTORY("${${PROJECT_NAME}_BINARY_DIR}/include/pinocchio/bindings/python/algorithm")
 MAKE_DIRECTORY("${${PROJECT_NAME}_BINARY_DIR}/include/pinocchio/bindings/python/utils")

